### DISCUSSION TITLE
SNZB-03, how to change reactivation time

### DISCUSSION THREAD

**Discussion by Daniel76145** [Koenkk/zigbee2mqtt#19387] [Devices]
Hi,

Is it possible to change reactivation time on the Sonoff PIR [SNZB-03](https://www.zigbee2mqtt.io/devices/SNZB-03.html#sonoff-snzb-03)? How?

**Comment by kkossev** [comment]
It is hardcoded in the firmware to 60 seconds, there is no way to change the reactivation time.
